src/main/java/me/earth/earthhack/impl/modules/render/popchams/PopChams.java

```diff
@@ -151,5 +160,13 @@ public double getZ() {
         public ModelPlayer getModel() {
             return model;
         }
+
+        private static ModelBiped.ArmPose getArmPose(EntityPlayer player, ItemStack stack) {
+            if (stack.isEmpty()) return ModelBiped.ArmPose.EMPTY;
+            System.out.println(player.getItemInUseCount());
```

Comment:
can we not debug here :)

Reply:
ohhh yeahhhh i forgot to remove this, sorry my bad